### ISSUE TITLE
Previous state of failed assertions carrying over

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#174]
If I save my spec (live reload) while the test runner is retrying a failing assertion, the test will appear to refresh in the Command Log, but continue to print the failing assertions. 

This started happening after `0.16.3` update in place of [`#73`](https://github.com/cypress-io/cypress/issues/173)

**Example Code:**
```
it "does not contain text", ->
  cy
     .visit('/')
     .get('body').contains('foo bar baz')
```

**Command Log after live reload**
<img width="566" alt="screen shot 2016-06-17 at 2 08 30 pm" src="https://cloud.githubusercontent.com/assets/1271364/16160224/008d505c-3495-11e6-9ae4-d4a3272d4347.png">

**Comment by brian-mann** [comment]
This is a regression from `0.16.2` when we added `regexp` support to `cy.contains`.

**Comment by brian-mann** [comment]
Fixed in [`0.16.4`](https://github.com/cypress-io/cypress/wiki/Changelog#0164-06172016).
